barserver/webpack.common.js

```diff
@@ -0,0 +1,20 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    app: './src/index.js',
```

Comment:
Don't think we have such file. 
```suggestion
    app: './src/index.ts',
```